keyboards/nightly_boards/daily60/config.h

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2022 DeskDaily
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 14
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS { 23, 24, 20, 19, 18 }
+#define MATRIX_COL_PINS { 22, 0, 1, 2, 5, 7, 8, 9, 10, 11, 12, 13, 14, 15 }
```

Comment:
These need to be `GP#` I believe.

Reply:
Hi @drashna. My pcb worked fine with this format. Should I change it to GP#?

Reply:
Ideally, yeah, the GP## notation should be used here.  Makes things easier to debug/etc.